packages/flutter/test/cupertino/radio_test.dart

```diff
@@ -428,4 +429,45 @@ void main() {
     // Release pointer after widget disappeared.
     await gesture.up();
   });
```

Comment:
Can you add a test that sets and verifies the mouse cursor when the user provides a WidgetStateProperty?

Reply:
Ah. I think I've run into an issue here.

It turns out that I can't even provide a `WidgetStateProperty<MouseCursor>` to the Material equivalent - only `MouseCursor`s are allowed. Should I

a) Refactor in `CupertinoRadio` so that `mouseCursor` is a `WidgetStateProperty<MouseCursor>` and not a `MouseCursor` (possibly doing the same in `Radio`) or

b) Have `mouseCursor` be a plain old `MouseCursor` here?

Reply:
I wonder if we have an existing plan to convert parameters like these to WidgetStateProperty, maybe @MitchellGoodwin knows? Should we be making stuff like this a WidgetStateProperty when we can?